app/work.html

```diff
@@ -134,11 +177,11 @@ <h1>Esri Canada</h1>
       <div class="container">
         <article class="row text-center">
           <div class="single-col">
-            <h1>Visual Sonics</h1>
-            <p class="lead skinny">FUJIFILM VisualSonics designs and manufactures ultra high frequency ultrasound imaging systems, for both research and clinical use.</p>
-            <img src="images/showcase-vs.png" alt="visual sonics mock-up" class="img-responsive">
+            <h1>HornBlower</h1>
```

Comment:
@derulski-therefore If you can please just adjust to be 'Hornblower', and do the build, PR will be good after that. Thanks!

Reply:
Outputted font style is block cap'd, so aok.